src/main/java/com/checkout/payments/previous/response/PaymentsQueryResponse.java

```diff
@@ -0,0 +1,23 @@
+package com.checkout.payments.previous.response;
+
+import com.checkout.HttpMetadata;
+import com.google.gson.annotations.SerializedName;
+import javax.validation.constraints.Size;
+import java.util.List;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public final class PaymentsQueryResponse extends HttpMetadata {
```

Comment:
This is the same as default src/main/java/com/checkout/payments/response/PaymentsQueryResponse.java so I think we can just leverage default for both